tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "include": ["remix.env.d.ts", "**/*.ts", "**/*.tsx"],
   "compilerOptions": {
-    "lib": ["DOM", "DOM.Iterable", "ES2019"],
+    "lib": ["DOM", "DOM.Iterable", "ES2019", "ES2021.String"],
```

Comment:
Will this affect both the browser build and the server build? Will it be polyfilled correctly for the browser?